resources/assets/js/components/JobBuilder/RedirectToLastIncompleteStep.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useEffect } from "react";
+import { useIntl } from "react-intl";
+import { Job, JobPosterKeyTask, Criteria } from "../../models/types";
+import {
+  criteriaProgressState,
+  jobTasksProgressState,
+  jobImpactProgressState,
+  jobBuilderEnvProgressState,
+  jobBuilderIntroProgressState,
+  jobBuilderDetailsProgressState,
+  JobBuilderPage,
+  VALID_COUNT,
+  pageToUrl,
+} from "./jobBuilderHelpers";
+import { ProgressTrackerState } from "../ProgressTracker/types";
+import { RootState } from "../../store/store";
+import {
+  getJob,
+  getTasksByJob,
+  getCriteriaByJob,
+  getJobIsUpdating,
+  getTasksForJobIsUpdating,
+  getCriteriaForJobIsUpdating,
+} from "../../store/Job/jobSelector";
+import { connect } from "react-redux";
+import JobBuilderStepContainer from "./JobBuilderStep";
+import { redirect } from "../../helpers/router";
+
+interface RedirectToLastIncompleteStepProps {
+  jobId: number;
+  job: Job | null;
+  jobIsLoading: boolean;
+  tasks: JobPosterKeyTask[];
+  maxTasksCount: number;
+  tasksIsLoading: boolean;
+  criteria: Criteria[];
+  criteriaIsLoading: boolean;
+  redirect: (url: string) => void;
+}
```

Comment:
Name `redirect` is already declared. Maybe try changing the prop to `redirectToUrl` to avoid confusion.

Reply:
I went with handleRedirect